src/libstd/sys/windows/ext/ffi.rs

```diff
@@ -42,11 +56,29 @@ impl OsStringExt for OsString {
 /// Windows-specific extensions to `OsStr`.
 #[stable(feature = "rust1", since = "1.0.0")]
 pub trait OsStrExt {
-    /// Re-encodes an `OsStr` as a wide character sequence,
-    /// i.e. potentially ill-formed UTF-16.
+    /// Re-encodes an `OsStr` as a wide character sequence, i.e. potentially
+    /// ill-formed UTF-16.
+    ///
+    /// This is lossless: calling [`OsString::from_wide()`] and then
```

Comment:
For my own curiosity, is there some discussion around why the decision was made to remove the `()` parens when referencing function in documentation? Personally I liked that style, so I'm curious as to why the change was made.